usecases/config/environment.go

```diff
@@ -679,6 +687,7 @@ const (
 	DefaultMaxConcurrentGetRequests            = 0
 	DefaultGRPCPort                            = 50051
 	DefaultMinimumReplicationFactor            = 1
+	DefaultMAXAllowedCollectionsCount          = 100
```

Comment:
100 is too small IMO. I would set it to 1k or even 2k

Reply:
we resolved it offline